### PR TITLE
refactor(action): adapt queue bytes

### PR DESCRIPTION
**Pull request by Kinplemelon**
https://emqx.atlassian.net/browse/ED-1596

https://emqx.atlassian.net/browse/ED-1657

### CODE REVIEW

File: src/common/tools.ts [hunk 1]
```diff
@@ -400,17 +400,17 @@ const ONE_MB = ONE_KB * 1024
...
-    return getNumPart(byte / ONE_KB) + 'KB'
+    return getNumPart(byte / ONE_KB) + ' KB'
   }
   if (byte < ONE_GB) {
-    return getNumPart(byte / ONE_MB) + 'MB'
+    return getNumPart(byte / ONE_MB) + ' MB'
   }
-  return getNumPart(byte / ONE_GB) + 'GB'
+  return getNumPart(byte / ONE_GB) + ' GB'
 }
```
Comment (ysfscream) [inline src/common/tools.ts:414]: I think it's good to add to shared-ui https://github.com/emqx/shared-ui/tree/main/packages/utils

File: src/i18n/RuleEngine.js [hunk 2]
```diff
@@ -1113,4 +1113,20 @@ export default {
...
+  buffer: {
+    zh: '缓冲',
+    en: 'Buffer',
+  },
+  queuedMessages: {
+    zh: '已缓存消息',
+    en: 'Queued Messages',
+  },
+  queueUsage: {
+    zh: '队列使用率',
```
Comment (ysfscream) [inline src/i18n/RuleEngine.js:1125]:
```suggestion
    zh: '队列使用量',
```

Is this not a “rate” here?

**Comment by ysfscream** [comment]
This pull request includes several improvements and new features related to metrics handling and display in the `src` directory. The most important changes involve the addition of new metrics functionalities, updates to existing components, and enhancements in data handling.

### Metrics Enhancements:

* Added a new slot for custom blocks in the `OverviewMetrics` component to display additional metrics data (`src/components/Metrics/OverviewMetrics.vue`).
* Introduced new event emitters for `metrics-change` and `node-change` in `OverviewMetrics` to handle real-time updates (`src/components/Metrics/OverviewMetrics.vue`). [[1]](diffhunk://#diff-f43ac16b86717f03b2117a22352fbfba4b9726b4fba59050c4c77cdbae6ed846R223-R226) [[2]](diffhunk://#diff-f43ac16b86717f03b2117a22352fbfba4b9726b4fba59050c4c77cdbae6ed846R245-R247) [[3]](diffhunk://#diff-f43ac16b86717f03b2117a22352fbfba4b9726b4fba59050c4c77cdbae6ed846R357)
* Implemented a new `useActionQueueMetrics` hook to manage and display queue metrics, including a line chart for queue bytes data (`src/hooks/useMetrics.ts`).

### Codebase Simplification:

* Replaced `toFixed` with `round` from `lodash` for better precision handling in `transMemorySizeNumToStr` function (`src/common/tools.ts`).
* Updated the `TypeMetricsItem` component to correctly calculate and display the difference in metrics data (`src/components/Metrics/TypeMetricsItem.vue`).

### New Features:

* Added new translations for buffer-related metrics in the i18n configuration (`src/i18n/RuleEngine.js`).
* Enhanced the `BridgeItemOverview` component to include new metrics for queued messages and queue usage, along with corresponding UI updates (`src/views/RuleEngine/Bridge/Components/BridgeItemOverview.vue`). [[1]](diffhunk://#diff-a4daf2201fae1dba774a0f05c462b3282445de71193ea281897e9c4ee43578a1R14-R71) [[2]](diffhunk://#diff-a4daf2201fae1dba774a0f05c462b3282445de71193ea281897e9c4ee43578a1R125-R137) [[3]](diffhunk://#diff-a4daf2201fae1dba774a0f05c462b3282445de71193ea281897e9c4ee43578a1R156-R157) [[4]](diffhunk://#diff-a4daf2201fae1dba774a0f05c462b3282445de71193ea281897e9c4ee43578a1R244-R278)

**Review by ysfscream — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-4884)

**Pull request merged**